packages/account/src/Styles/account.scss

```diff
@@ -1447,7 +1450,11 @@ $MIN_HEIGHT_FLOATING: calc(
                     }
                     &--small {
                         max-width: 99px;
-
+                        
+                        .data-list__row-content .dc-btn {
+                            left: 50%;
+                            transform: translateX(-52%);
+                        }
```

Comment:
Why do we need this change on mobile mode? It looks fine for me without this change

Reply:
We have to make this change in order to fit the revoke access button in the column (so that it doesn't get truncated) when we switch to other languages like: ES, FR, ID, IT, RU - in mobile view.

Reply:
Alright. Got it